crates/api_common/src/utils.rs

```diff
@@ -835,13 +840,101 @@ fn limit_expire_time(expires: DateTime<Utc>) -> LemmyResult<Option<DateTime<Utc>
   }
 }
 
+pub async fn process_markdown(
+  text: &str,
+  slur_regex: &Option<Regex>,
+  context: &LemmyContext,
+) -> LemmyResult<String> {
+  let text = remove_slurs(text, slur_regex);
+  let (text, links) = markdown_rewrite_image_links(text);
```

Comment:
It should only rewrite these image links if proxy_image is true, correct?

Reply:
Yes it checks the setting inside of markdown_rewrite_image_links